ERROR - 2024-01-05 08:11:01 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: YES) in C:\xampp\htdocs\c4\system\Database\MySQLi\Connection.php:184
Stack trace:
#0 C:\xampp\htdocs\c4\system\Database\MySQLi\Connection.php(184): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), '', 3306, '', 0)
#1 C:\xampp\htdocs\c4\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\c4\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\c4\system\Database\BaseBuilder.php(2302): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\c4\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\c4\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\c4\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\c4\app\Controllers\registration.php(39): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\c4\system\CodeIgniter.php(941): App\Controllers\Registration->process()
#9 C:\xampp\htdocs\c4\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registration))
#10 C:\xampp\htdocs\c4\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\c4\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\c4\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\c4\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\c4\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\c4\system\Database\BaseBuilder.php(2302): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#3 C:\xampp\htdocs\c4\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#4 C:\xampp\htdocs\c4\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#5 C:\xampp\htdocs\c4\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, true)
#6 C:\xampp\htdocs\c4\app\Controllers\registration.php(39): CodeIgniter\Model->insert(Array)
#7 C:\xampp\htdocs\c4\system\CodeIgniter.php(941): App\Controllers\Registration->process()
#8 C:\xampp\htdocs\c4\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registration))
#9 C:\xampp\htdocs\c4\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\c4\public\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-01-05 08:11:01 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2302): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`username`, `email`, `password`) VALUES (:username:, :email:, :password:)', [...], false)
 3 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 4 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 5 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], true)
 6 APPPATH\Controllers\registration.php(39): CodeIgniter\Model->insert([...])
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Registration->process()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registration))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-05 08:24:37 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: YES) in C:\xampp\htdocs\c4\system\Database\MySQLi\Connection.php:184
Stack trace:
#0 C:\xampp\htdocs\c4\system\Database\MySQLi\Connection.php(184): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), '', 3306, '', 0)
#1 C:\xampp\htdocs\c4\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\c4\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\c4\system\Database\BaseBuilder.php(2302): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\c4\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\c4\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\c4\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\c4\app\Controllers\registration.php(39): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\c4\system\CodeIgniter.php(941): App\Controllers\Registration->process()
#9 C:\xampp\htdocs\c4\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registration))
#10 C:\xampp\htdocs\c4\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\c4\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\c4\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\c4\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\c4\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\c4\system\Database\BaseBuilder.php(2302): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#3 C:\xampp\htdocs\c4\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#4 C:\xampp\htdocs\c4\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#5 C:\xampp\htdocs\c4\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, true)
#6 C:\xampp\htdocs\c4\app\Controllers\registration.php(39): CodeIgniter\Model->insert(Array)
#7 C:\xampp\htdocs\c4\system\CodeIgniter.php(941): App\Controllers\Registration->process()
#8 C:\xampp\htdocs\c4\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registration))
#9 C:\xampp\htdocs\c4\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\c4\public\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-01-05 08:24:37 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2302): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`username`, `email`, `password`) VALUES (:username:, :email:, :password:)', [...], false)
 3 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 4 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 5 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], true)
 6 APPPATH\Controllers\registration.php(39): CodeIgniter\Model->insert([...])
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Registration->process()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registration))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-05 08:30:20 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: YES) in C:\xampp\htdocs\c4\system\Database\MySQLi\Connection.php:184
Stack trace:
#0 C:\xampp\htdocs\c4\system\Database\MySQLi\Connection.php(184): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), '', 3306, '', 0)
#1 C:\xampp\htdocs\c4\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\c4\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\c4\system\Database\BaseBuilder.php(2302): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\c4\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\c4\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\c4\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\c4\app\Controllers\registration.php(39): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\c4\system\CodeIgniter.php(941): App\Controllers\Registration->process()
#9 C:\xampp\htdocs\c4\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registration))
#10 C:\xampp\htdocs\c4\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\c4\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\c4\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\c4\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\c4\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\c4\system\Database\BaseBuilder.php(2302): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#3 C:\xampp\htdocs\c4\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#4 C:\xampp\htdocs\c4\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#5 C:\xampp\htdocs\c4\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, true)
#6 C:\xampp\htdocs\c4\app\Controllers\registration.php(39): CodeIgniter\Model->insert(Array)
#7 C:\xampp\htdocs\c4\system\CodeIgniter.php(941): App\Controllers\Registration->process()
#8 C:\xampp\htdocs\c4\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registration))
#9 C:\xampp\htdocs\c4\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\c4\public\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-01-05 08:30:20 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2302): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`username`, `email`, `password`) VALUES (:username:, :email:, :password:)', [...], false)
 3 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 4 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 5 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], true)
 6 APPPATH\Controllers\registration.php(39): CodeIgniter\Model->insert([...])
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Registration->process()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registration))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
